movement/io/load_poses.py

```diff
@@ -227,6 +229,12 @@ def _load_from_sleap_analysis_file(
     -------
     movement.io.tracks_validators.ValidPoseTracks
         The validated pose tracks and confidence scores.
+
+    Notes
+    -----
+    If the point-level prediction scores (i.e. confidence) in the SLEAP
+    analysis file are all NaNs, this function assumes that the pose
+    tracks are user-labeled, and will assign a fixed score of 1.0.
```

Comment:
I agree with making this assumption, but could be risky to do this "silently". Should we log a warning when we do this? Both here and for the labels file.

Reply:
Good idea! Will do that.